exec/java-exec/src/main/java/org/apache/drill/exec/rpc/data/DataClientConnection.java

```diff
@@ -84,8 +85,8 @@ public boolean equals(Object obj) {
     return true;
   }
 
-  public void shutdownIfClient() {
-    Closeables.closeQuietly(client);
+  public void shutdownIfClient() throws IOException {
```

Comment:
Same comment as above, re: either need to declare or catch IOException, despite the fact that ```Closeables.close()``` may never actually throw IOE.